src/lib/crypto/kimchi_bindings/stubs/src/pasta_fp_plonk_index.rs

```diff
@@ -38,6 +38,7 @@ impl ocaml::custom::Custom for CamlPastaFpPlonkIndex {
 pub fn caml_pasta_fp_plonk_index_create(
     gates: CamlPastaFpPlonkGateVectorPtr,
     public: ocaml::Int,
+    prev_challenges: ocaml::Int,
```

Comment:
I think we should use `usize` and let ocaml-rs do the conversion here :o

Reply:
this would make the rest of the code cleaner as you would not need all the conversions